### PR TITLE
docs: add trg 2-4 leading readme example

### PR DESCRIPTION
**Pull request by FaGru3n**
## Description
- add example for leading repo readme
- add link to Readme TRG 1-01

fixes https://github.com/eclipse-tractusx/sig-infra/issues/370

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by almadigabor — APPROVED** [review]
LGTM

**Review by hzierer — APPROVED** [review]
LGTM

File: docs/release/trg-2/trg-2-4.md [hunk 1]
```diff
@@ -2,29 +2,38 @@
...
+| Status | Created     | Post-History                                   |
+|--------|-------------|------------------------------------------------|
+| Update | 12-Dez-2023 | add example leading repository reference links |
+| Active | 07-Mar-2023 |                                                |
+|        | 10-Jan-2023 | Initial release                                |
 
 ## Why
 
-We have multiply stakeholders for a repository: System team, release management, test management, external people who are trying it out, want to contribute or need to set it up.
+We have multiply stakeholders for a repository: system team, release management, test management, external people who are trying it out, want to contribute or need to set it up.
```
Comment (SebastianBezold) [inline docs/release/trg-2/trg-2-4.md:13]: multiple instead of multiply 😁 was already wrong in the original

File: docs/release/trg-2/trg-2-4.md [hunk 2]
```diff
@@ -2,29 +2,38 @@
...
 They need to be able to understand complicated product structures in an intuitive way to have an easier task of diving into a product.
 
 ## Description
 
-Each product **must** have a leading product repository that can point the users to all the parts/components/charts/releases of that product.
+Each product **must** have a leading product repository that points to all the parts/components/charts/releases of the product.
 This can be achieved by having a single repository for a product or creating references to the other repositories.
 
-A good example when a product has a separate repository for frontend and backend. Then the product team **must** have a leading repository where they reference the components/repositories of the product:
+Good examples of products got separate repositories for each component (e.g. frontend and backend).
```
Comment (SebastianBezold) [inline docs/release/trg-2/trg-2-4.md:21]: Not sure, what this sentence should tell me. I think we don't lose much, if we keep this one out.

File: docs/release/trg-2/trg-2-4.md [hunk 3]
```diff
@@ -2,29 +2,38 @@
...
+
+The product **must** have a leading repository where they reference the components/repositories of the product:
```
Comment (SebastianBezold) [inline docs/release/trg-2/trg-2-4.md:23]: Duplicate to the first sentence in "## Description"

**Review by SebastianBezold — CHANGES_REQUESTED** [review]

File: docs/release/trg-2/trg-2-4.md (hunk 2 above)
Comment (stephanbcbauer) [reply docs/release/trg-2/trg-2-4.md:21]: i think the examples are missing here?

**Review by stephanbcbauer — APPROVED** [review]
I think the changes themselves are good and appropriate, but ... will contact you because of the TRG itself :)

File: docs/release/trg-2/trg-2-4.md (hunk 3 above)
Comment (FaGru3n) [reply docs/release/trg-2/trg-2-4.md]: [6d1bdb2](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/551/commits/6d1bdb2ddb0f607fae53a875d68eb018db5809be)

File: docs/release/trg-2/trg-2-4.md (hunk 1 above)
Comment (FaGru3n) [reply docs/release/trg-2/trg-2-4.md]: [6d1bdb2](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/551/commits/6d1bdb2ddb0f607fae53a875d68eb018db5809be)
done

File: docs/release/trg-2/trg-2-4.md [hunk 4]
```diff
@@ -2,29 +2,34 @@
...
+| Status | Created     | Post-History                                   |
+|--------|-------------|------------------------------------------------|
+| Update | 12-Dez-2023 | add example leading repository reference links |
+| Active | 07-Mar-2023 |                                                |
+|        | 10-Jan-2023 | Initial release                                |
 
 ## Why
 
-We have multiply stakeholders for a repository: System team, release management, test management, external people who are trying it out, want to contribute or need to set it up.
+We have multiple stakeholders for a repository: system team, release management, test management, external people who are trying it out, want to contribute or need to set it up.
```
Comment (hzierer) [inline docs/release/trg-2/trg-2-4.md:13]: external people? from what?  how about ..., interested persons trying it out, ...

File: docs/release/trg-2/trg-2-4.md [hunk 5]
```diff
@@ -2,29 +2,34 @@
...
-Each product **must** have a leading product repository that can point the users to all the parts/components/charts/releases of that product.
+Each product **must** have a leading product repository that points to all the parts/components/charts/releases of the product.
 This can be achieved by having a single repository for a product or creating references to the other repositories.
 
-A good example when a product has a separate repository for frontend and backend. Then the product team **must** have a leading repository where they reference the components/repositories of the product:
+Exemplary repository structure:
 
 - productname: leading product repository
-  - README.md: contains the urls for the backend and frontend applications
+  - README.md: contains the urls for e.g. the backend and frontend applications
```
Comment (hzierer) [inline docs/release/trg-2/trg-2-4.md:24]: urls -> URLs  (btw, multiple times in the Text)

File: docs/release/trg-2/trg-2-4.md [hunk 6]
```diff
@@ -2,29 +2,34 @@
...
+  - README.md: each repo follows our [TRG 1.01](../trg-1/trg-1-1.md) about the README.md
```
Comment (hzierer) [inline docs/release/trg-2/trg-2-4.md:25]: has to follow TRG 1.01 about README.md?

**Review by hzierer — CHANGES_REQUESTED** [review]

File: docs/release/trg-2/trg-2-4.md (hunk 6 above)
Comment (FaGru3n) [reply docs/release/trg-2/trg-2-4.md]: [9730695](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/551/commits/9730695e42a8c04c8343391c9012f08863750815)

File: docs/release/trg-2/trg-2-4.md (hunk 5 above)
Comment (FaGru3n) [reply docs/release/trg-2/trg-2-4.md]: [9730695](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/551/commits/9730695e42a8c04c8343391c9012f08863750815)

File: docs/release/trg-2/trg-2-4.md (hunk 4 above)
Comment (FaGru3n) [reply docs/release/trg-2/trg-2-4.md]: [9730695](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/551/commits/9730695e42a8c04c8343391c9012f08863750815)

File: docs/release/trg-2/trg-2-4.md [hunk 7]
```diff
@@ -2,34 +2,47 @@
...
+| Status | Created     | Post-History                                   |
+|--------|-------------|------------------------------------------------|
+| Update | 12-Dez-2023 | add example leading repository reference links |
+| Active | 07-Mar-2023 |                                                |
+|        | 10-Jan-2023 | Initial release                                |
 
 ## Why
 
-We have multiply stakeholders for a repository: System team, release management, test management, external people who are trying it out, want to contribute or need to set it up.
+We have multiple stakeholders for a repository: system team, release management, test management, interested persons trying it out, want to contribute or need to set it up.
```
Comment (hzierer) [inline docs/release/trg-2/trg-2-4.md:13]:
```suggestion
We have multiple stakeholders for a repository: system team, test management, release management and everybody else interested in the product.
```

**Review by hzierer — CHANGES_REQUESTED** [review]
added one suggestion, the rest LGTM.

**Review by almadigabor — APPROVED** [review]
LGTM

**Pull request merged**